components/Receiver.js

```diff
@@ -18,7 +18,7 @@ class Receiver extends Component {
   componentDidMount = async() => {
     await this.updateTimes()
   }
-  
+
```

Comment:
I made a comment of this in the other PR; this looks like the work of an auto linter. I'm not familiar enough with them, but logically thinking, I think this "could" be contributing to the problems Naomi kept running into. I probably wouldn't jump into it as a first path to fix an error, or even a second, but if you continue to have problems maybe it would eventually be a place to look at.